src/core/installAddon.js

```diff
@@ -481,13 +516,19 @@ export class WithInstallHelpers extends React.Component {
       })
       .catch((error) => {
         log.error(`Install error: ${error}`);
+
         dispatch(
           setInstallState({
             guid,
             status: ERROR,
             error: FATAL_INSTALL_ERROR,
           }),
         );
+        _tracking.sendEvent({
+          action: getAddonTypeForTracking(type),
+          category: getAddonEventCategory(type, INSTALL_ERRORED_ACTION),
+          label: name,
```

Comment:
should we also sendEvent that install_failed_action? I am still not super clear on which one to use if you are choosing one over the other